app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -59,6 +59,7 @@ def clear_session
 
   # Destroys the users session in 1) Redis, 2) the MHV SSO Cookie, 3) and the Session Cookie
   def reset_session
+    AfterLogoutJob.perform_async(account_uuid: @current_user&.account_uuid)
```

Comment:
Should we put the `unless Rails.env.production?` on this line? Shouldn't be a big deal since they'll just cancel out, but we'll queue up a lot of Sidekiq jobs on prod that won't do anything

Reply:
The thinking was that we should make the new `AfterLogoutJob` worker mirror the existing `AfterLoginJob` as closely as possible and put the environment check within it to allow for future logic to be placed within the worker; I suppose since it just has the TUD function right now we should move it and leave it to whoever ends up adding more functionality to the `AfterLogoutJob` in the future to put it back where it is now.